.github/workflows/cmake.yml

```diff
@@ -75,6 +75,9 @@ jobs:
         include:
           - cc: gcc
             cxx: g++
+          - cc: gcc
+            cxx: g++
+            apache-arrow: "main"
```

Comment:
Can we use better name for this?

We just need a boolean value not a branch name, right?

For example, PGroonga uses `groonga-main: yes` and `postgresql-unreleased: yes`: https://github.com/pgroonga/pgroonga/blob/587011468479a153cd0eb60d9adc9f034703e082/.github/workflows/linux.yml#L44-L49

If we want to use a branch name here, we should use it in `actions/checkout` for apache/arrow.

Reply:
fix: [33a428f](https://github.com/groonga/groonga/pull/2459/commits/33a428fb47575a588e36c865bf739e342ec7bc04)